### PR TITLE
Fix color contrast issue

### PR DESCRIPTION
**Pull request by shunguoy**
#826

### CODE REVIEW

File: accessibility-checker-engine/src/v2/checker/accessibility/util/legacy.ts [hunk 1]
```diff
@@ -3178,22 +3178,27 @@ export class RPTUtil {
...
             "hasBGImage": false,
             "fg": null,
             "bg": null
         };
-        var compStyleColor = win.getComputedStyle(ruleContext).color;
+        
+        // start
+        var cStyle = win.getComputedStyle(ruleContext);
+        var compStyleColor = cStyle.color;
+        var compStyleBgColor = cStyle.backgroundColor;
```
Comment (tombrunet) [inline accessibility-checker-engine/src/v2/checker/accessibility/util/legacy.ts:3196]: Doesn't seem this additional line is used?

File: accessibility-checker-engine/src/v2/checker/accessibility/util/legacy.ts [hunk 2]
```diff
@@ -3178,22 +3178,27 @@ export class RPTUtil {
...
         if (!compStyleColor)
             compStyleColor = "black";
         var fg = RPTUtil.Color(compStyleColor);
-
+        var bg = RPTUtil.Color(compStyleBgColor);
```
Comment (tombrunet) [inline accessibility-checker-engine/src/v2/checker/accessibility/util/legacy.ts:3200]: Doesn't seem this additional line is used?

File: accessibility-checker-engine/src/v2/checker/accessibility/util/legacy.ts (hunk 1 above)
Comment (shunguoy) [reply accessibility-checker-engine/src/v2/checker/accessibility/util/legacy.ts:3196]: removed.

File: accessibility-checker-engine/src/v2/checker/accessibility/util/legacy.ts (hunk 2 above)
Comment (shunguoy) [reply accessibility-checker-engine/src/v2/checker/accessibility/util/legacy.ts:3200]: removed

**Review by eirckr — APPROVED** [review]

**Pull request merged**